Events/MessageEvent.cs

```diff
@@ -142,19 +142,23 @@ public static async Task MessageHandlerAsync(DiscordClient client, MockDiscordMe
                 {
                     if (Program.db.SetContains("trackedUsers", message.Author.Id))
                     {
-                        DiscordThreadChannel relayThread;
-
-                        if (trackingThreadCache.ContainsKey(message.Author.Id))
-                        {
-                            relayThread = trackingThreadCache[message.Author.Id];
+                        // Check current channel against tracking channels
+                        var trackingChannels = await Program.db.HashGetAsync("trackingChannels", message.Author.Id);
+                        var trackingChannelsList = JsonConvert.DeserializeObject<List<ulong>>(trackingChannels);
```

Comment:
This needs a null check on `trackingChannels`

Reply:
Done in https://github.com/Erisa/Cliptok/pull/236/commits/d64550572049108f070d535cc4c71a22431e25df